modules/op-release-notes-1-11.adoc

```diff
@@ -0,0 +1,198 @@
+** Replace the context of a pipeline task with another context in the `matrix.include` configuration.
+
+* With this update, the `TaskRun` resource validation process also validates the `matrix.include` parameters. The validation checks whether all parameters have values and match the specified type, and object parameters have all the keys required.
+
+* This update adds a new `default-resolver-type` field in the `default-configs` config map. You can set the value of this field to configure a default resolver.
+
+* With this update, you can define and use a `PipelineRun` context variable in the `pipelineRun.workspaces.subPath` configuration.
+
+* With this update, the `ClusterResolver`, `BundleResolver`, `HubResolver`, and `GitResolver` features are now available by default. 
+
+
+[id="triggers-new-features-1-11_{context}"]
+=== Triggers
+
+* With this update, Tekton Triggers support the `Affinity` and `TopologySpreadConstraints` values in the `EventListener` specification. You can use these values to configure Kubernetes and custom resources for an `EventListener` object.
+* This update adds a Slack interceptor that allows you to extract fields by using a slash command in Slack. The extracted fields are sent in the form data section of an HTTP request.
+
+
+[id="operator-new-features-1-11_{context}"]
+=== Operator
+
+* With this update, you can configure pruning for each `PipelineRun` or `TaskRun` resource by setting a `prune-per-resource` boolean field in the `TektonConfig` CR. You can also configure pruning for each `PipelineRun` or `TaskRun` resource in a  namespace by adding the `operator.tekton.dev/prune.prune-per-resource=true` annotation to that namespace.
+
+* With this update, if there are any changes in the {product-title} cluster-wide proxy, Operator Lifecycle Manager (OLM) recreates the {pipelines-title} Operator. 
+* With this update, you can disable the pruner feature by setting the value of the `config.pruner.disabled` field to `true` in the `TektonConfig` CR.
+
+
+[id="chains-new-features-1-11_{context}"]
+=== {tekton-chains}
+
+* With this update, {tekton-chains} is now generally available for use.
+
+* With this update, you can use the skopeo tool with {tekton-chains} to generate keys, which are used in the `cosign` signing scheme. 
+
+
+[id="tekton-hub-new-features-1-11_{context}"]
+=== {tekton-hub}
+
+:FeatureName: {tekton-hub}
+include::snippets/technology-preview.adoc[]
+
+* This update adds a new `resource/<catalog_name>/<kind>/<resource_name>/raw` endpoint and a new `resourceURLPath` field in the `resource` API response. This update helps you to obtain the latest raw YAML file of the resource.
+
+[id="tekton-results-new-features-1-11_{context}"]
+=== Tekton Results
+
+:FeatureName: Tekton Results
+include::snippets/technology-preview.adoc[]
+
+* This update adds Tekton Results to the Tekton Operator as an optional component.
+
```

Comment:
Can you add installation instructions:

To install Tekton Results, please use following commands:
```
export NAMESPACE="openshift-pipelines"
# Create DB user and password
oc create secret generic tekton-results-postgres --namespace=${NAMESPACE} --from-literal=POSTGRES_USER=result --from-literal=POSTGRES_PASSWORD=$(openssl rand -base64 20)

# Generate new self-signed cert or ca provided
openssl req -x509 \
-newkey rsa:4096 \
-keyout key.pem \
-out cert.pem \
-days 365 \
-nodes \
-subj "/CN=tekton-results-api-service.${NAMESPACE}.svc.cluster.local" \
-addext "subjectAltName = DNS:tekton-results-api-service.${NAMESPACE}.svc.cluster.local"

# Create new TLS Secret from cert.
oc create secret tls -n ${NAMESPACE} tekton-results-tls \
--cert=cert.pem \

# Create PVC for logging if PVC storage for logging is required
cat <<EOF > pvc.yaml
apiVersion: v1
kind: PersistentVolumeClaim
metadata:
  name: tekton-logs
  namespace: tekton-pipelines
spec:
  accessModes:
  - ReadWriteOnce
  resources:
    requests:
      storage: 1Gi
EOF
// Apply the above PVC
oc apply -f pvc.yaml
```
Now apply the TektonResult CR needed:
```
apiVersion: operator.tekton.dev/v1alpha1
kind: TektonResult
metadata:
  name: result
spec:
  targetNamespace: openshift-pipelines
  logs_api: true
  log_level: debug
  db_port: 5432
  db_user: result
  db_host: tekton-results-postgres-service.openshift-pipelines.svc.cluster.local
  logging_pvc_name: tekton-logs
  logs_path: /logs
  logs_type: File
  logs_buffer_size: 32768
  auth_disable: true
  tls_hostname_override: tekton-results-api-service.openshift-pipelines.svc.cluster.local
  db_enable_auto_migration: true
  server_port: 8080
  prometheus_port: 9090
  ```